src/api/sshKeys.ts

```diff
@@ -0,0 +1,14 @@
+import express from 'express'
+import fs from 'fs'
+const sshKeyRouter = express.Router()
+
+sshKeyRouter.get('/', (req, res) => {
+  //res.json([])
+  fs.readFile('/home/noob101/.ssh/authorized_keys', (error, data) => {
```

Comment:
```
const os = require('os');

console.log(os.homedir());
```
https://stackoverflow.com/questions/9080085/node-js-find-home-directory-in-platform-agnostic-way